loc/en.js

```diff
@@ -382,6 +382,7 @@ module.exports = {
       address: 'address',
       address_field_is_not_valid: 'Address field is not valid',
       amount_field_is_not_valid: 'Amount field is not valid',
+      amount_is_too_heigh: "There is a maximum of 21,000,000 BTCV that will ever exist, so don't try to send more!",
```

Comment:
it should be `high` instead of `heigh`, I guess

Reply:
+1 :)